scripts/test

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 
-npm test && npx nyc report
+npm run test:unit
+npm run test:ct-unit
+npm start & wait-on http://localhost:3000
```

Comment:
I think here you mean `&&`?

Reply:
I guess this file no longer exists?

Reply:
> I guess this file no longer exists?

Exactly this file exists no more, I split the unit and integration tests.